.github/workflows/test.yml

```diff
@@ -71,13 +71,7 @@ jobs:
             - name: Install xvfb on Linux
               if: runner.os == 'Linux'
               run: sudo apt-get update && sudo apt-get install -y xvfb
-
-            - name: Install local modules on windows
```

Comment:
So biome works directly in windows?

Reply:
Yup. This step was only run for building the eslint custom plugin, which has been removed in this PR